xo-update.sh

```diff
@@ -13,6 +13,8 @@ echo $output updates available
 if [[ $output -ne 0 ]]; then
   echo "Updating from source..."
   sudo git pull
+  sudo rm -rf dist
+  sudo rm -rf node_modules
```

Comment:
Removing the node_modules results in the need to reinstall any custom modules that may have been installed. For example the email plugin to get notified of the backup job status.